engine/plugins/client/client.js

```diff
@@ -6,37 +6,90 @@
 'use strict';
 
 const debug = require('debug')('bibbox:CLIENT:main');
-const clients = {};
+const redis = require('redis');
+const Q = require('q');
 
 /**
  * The Client object.
  *
+ * @param redisConfig
+ *   Redis configuration.
+ * @param persistent
+ *   If true redis will be use else in-memory store (mostly useful for testing).
```

Comment:
"If true redis will be use else in-memory store (mostly useful for testing)."
=>
"If true redis will be used. Otherwise in-memory store will be used (mostly useful for testing)."

Reply:
Fixed